django/db/models/query.py

```diff
@@ -416,6 +416,8 @@ def get(self, *args, **kwargs):
         Perform the query and return a single object matching the given
         keyword arguments.
         """
+        if args or kwargs:
+            self._not_support_combined_queries('get')
```

Comment:
I think we should use a separate message because the current can be misleading, e.g.:
```suggestion
        if self.query.combinator and (args or kwargs):
            raise NotSupportedError(
                'Calling QuerySet.get(...) with filters after %s() is not '
                'supported.' % self.query.combinator
            )
```